CONTRIBUTING.md

```diff
@@ -18,11 +18,13 @@ If you encounter a problem with someone's conduct, contact the prpl Community Ma
 
 ## Reporting bugs
 
-To report bugs, use the [github issues](https://github.com/prplfoundation/prplMesh/issues/new/choose) entry form.
+To report bugs, use the [JIRA create issue](https://jira.prplfoundation.org/secure/CreateIssue!default.jspa) entry form.
+Select Prplmesh as the project, and "Bug" as the issue type (you can use "Task" for feature requests)
 
 ## Improving documentation
 
 High-level developer documentation can be found on the [wiki](https://github.com/prplfoundation/prplMesh/wiki), which can be edited in-place.
+This is being migrated to [Confluence](https://confluence.prplfoundation.org/display/PRPLMESH), so please check there first.
```

Comment:
This page is not publicly accessible yet.
It also seems the accounts are not linked with Jira, because I don't have access to it either and it refuses my login+password.

Reply:
Maybe remove this part of the PR until Confluence is really active.

Reply:
We can always add this mention quite quick. So i don't see a reason to add link to this page while it's not available .

Reply:
Ok, I've omitted it